toggl/base.py

```diff
@@ -60,7 +60,12 @@ def __init__(self, api, **kwargs):
         self._update_attrs(kwargs)
 
     def _update_attrs(self, attrs):
-        for k, v in attrs.iteritems():
+        try:
```

Comment:
To avoid checking for items/iteritems everywhere, let's just use `items` everywhere (it works in both 2 and 3, and for the amount of data that's likely to be handled here, just as well).